experimental/packages/otlp-exporter-base/src/OTLPExporterBase.ts

```diff
@@ -117,15 +117,20 @@ export abstract class OTLPExporterBase<
     return this._shutdownOnce.call();
   }
 
+  /**
+   * Exports any pending spans in the exporter
+   */
+  forceFlush(): Promise<void> {
+    return Promise.all(this._sendingPromises).then(() => {});
```

Comment:
Why the additional `then(() => {});` as the Promise.all returns a promise anyway.

Reply:
If it's for the previous comment `/** ignore resolved values */` then please re-include this comment

Reply:
@JacksonWeber you marked this as resolved but I still don't see the comment

Reply:
Nevermind I was looking at an old version